src/store/metadata/stub/content/editions/versions/aged-19-years-and-over-further-education-and-skills-learner-achievements_2023-05.json

```diff
@@ -0,0 +1,125 @@
+{
+    "@context": "https://staging.idpd.uk/ns#",
+    "@id": "https://staging.idpd.uk/datasets/aged-19-years-and-over-further-education-and-skills-learner-achievements/editions/2023-05/versions",
+    "@type": "hydra:Collection",
+    "title": "Aged 19 years and over further education and skills learner achievements",
+    "versions": [
+        {
+            "@type": [
+              "dcat:Distribution",
+              "csvw:Table"
+            ],
+            "@id": "https://staging.idpd.uk/datasets/aged-19-years-and-over-further-education-and-skills-learner-achievements/editions/2023-05/versions/1",
+            "identifier": "1",
+            "issued": "2023-05-17T12:41:00Z",
+            "title": "Aged 19 years and over further education and skills learner achievements",
+            "summary": "Count of learners achieving a funded further education and skills learning aim, with the exception of community learning, non-qualification parts of the Multiply programme and bootcamps, England, academic years 2016/17 to 2021/22",
+            "description": "This dataset shows the count of learners achieving a funded further education and skills learning aim, with the exception of community learning, non-qualification parts of the Multiply programme and bootcamps, in England, for academic years 2016/17 to 2021/22.\n\nCoverage \n\nEngland\n\nGeography definition\n\nCounties and Unitary Authorities,\nNation (England)\n\nSource\n\n - England: [Department for Education]",
```

Comment:
To avoid repetition of requested changes, please effect all the changes suggested for the editions file (as individually listed above) on the versions file as well.